purchase_procurement_analytic/models/stock.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Carlos Dauden <carlos.dauden@tecnativa.com>
+# Copyright 2017 Tecnativa - Vicent Cubells <vicent.cubells@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import api, models
+
+
+class StockMove(models.Model):
+    _inherit = 'stock.move'
+
+    @api.model
+    def _prepare_procurement_from_move(self):
```

Comment:
@cubells It seems that this piece of code should have been placed in a more generic module 'stock_procurement_analytic' for example (it does not imply purchases necessarily...).

What do you think about it ?